src/phone-input/stateful-phone-input-container.js

```diff
@@ -58,9 +57,10 @@ export default class StatefulPhoneInputContainer extends React.Component<
 
   render() {
     const {children, initialState, stateReducer, ...restProps} = this.props;
+    // $FlowFixMe
```

Comment:
please remove this - the flow refactor is unsafe if we start introducing ignores where previously not required